src/compiler/emitter.ts

```diff
@@ -1257,26 +1257,34 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
 
                     // Children
                     if (children) {
-                        for (let i = 0; i < children.length; i++) {
-                            // Don't emit empty expressions
-                            if (children[i].kind === SyntaxKind.JsxExpression && !((<JsxExpression>children[i]).expression)) {
-                                continue;
+                        const isChildEmittable = (child: JsxChild): boolean => {

```

Comment:
i do nto think we need this function, but even if we did we should not create one every time we enter this block.